pkg/eventing/v1alpha1/client.go

```diff
@@ -163,6 +163,14 @@ func (b *TriggerBuilder) Subscriber(subscriber *duckv1.Destination) *TriggerBuil
 // Broker to set the broker of trigger object
 func (b *TriggerBuilder) Broker(broker string) *TriggerBuilder {
 	b.trigger.Spec.Broker = broker
+
+	return b
+}
+
+// InjectBroker to add annotation to setup default broker
+func (b *TriggerBuilder) InjectBroker() *TriggerBuilder {
```

Comment:
final request: Could we make it taking a boolean arg and enable it when its true, otherwise delete it ? This is useful if you hand over builders or init them from an existing trigger.

Reply:
Sure, sounds useful.